### ISSUE TITLE
Verify kodando-react library completion

### ISSUE THREAD

**Issue by danfma** [kodando/kodando#18]

**Comment by felipehjcosta** [comment]
I tried it in a pet project and noticed that the documentation seems outdated. WDYT about updating them?

**Comment by danfma** [comment]
Yes, the documentation isn't up to date. We need to update it. I'm adding more one library (mobx) and I will put more samples in the kodando-samples.
